### PR TITLE
[one-cmds] Add fake quantization option to one-quantize

### PR DESCRIPTION
**Pull request by jinevening**
This adds fake quantization option to one-quantize.

ONE-DCO-1.0-Signed-off-by: Hyukjin Jeong <hj1.jeong@samsung.com>

### CODE REVIEW

**Comment by jinevening** [comment]
To reviewers: This PR allows users (ex: python interface) to use fake quantization feature.

**Review by llFreetimell — APPROVED** [review]
LGTM

File: compiler/one-cmds/one-quantize [hunk 1]
```diff
@@ -512,6 +525,27 @@ def _copy_qparam(args):
...
+    logfile_path = os.path.realpath(args.output_path) + '.log'
+
+    with open(logfile_path, 'wb') as f:
+        # get driver path
+        circle_quantizer_path = os.path.join(dir_path, 'circle-quantizer')
+        q_model = ""
+        if _utils._is_valid_attr(args, 'input_path'):
+            q_model = getattr(args, 'input_path')
+        if _utils._is_valid_attr(args, 'output_path'):
+            fq_model = getattr(args, 'output_path')
```
Comment (seanshpark) [inline compiler/one-cmds/one-quantize:540]: I think we don't have to call `if _utils._is_valid_attr():` for input_path nor output_path as these are checked at `_verify_arg()`

there are more codes that checks again in this file and would be, in another PR, OK to remove them.

**Review by llFreetimell — APPROVED** [review]
LGTM

**Pull request merged**